app/src/main/res/layout/activity_chat.xml

```diff
@@ -4,37 +4,42 @@
     android:id="@+id/coordinator_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/default_bg">
+    android:background="@color/md_white_1000">
 
     <RelativeLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         app:layout_behavior="org.fossasia.susi.ai.helper.SnackbarBehavior"
         tools:context=".chat.ChatActivity">
 
-        <android.support.v7.widget.RecyclerView
-            android:id="@+id/rv_chat_feed"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:layout_above="@+id/sendMessageLayout"
-            android:layout_alignParentTop="true"
-            android:scrollbars="vertical" />
-
         <EditText
-            android:layout_width="match_parent"
             android:id="@+id/chatSearchInput"
-            android:textColor="@color/md_white_1000"
-            android:layout_height="40dp"
-            android:layout_marginRight="@dimen/margin_extremely_large"
-            android:hint="Search Chat Messages"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/height_large"
             android:layout_marginLeft="@dimen/margin_medium"
             android:layout_marginTop="@dimen/margin_medium"
-            android:visibility="invisible"
+            android:layout_marginRight="@dimen/margin_extremely_large"
+            android:hint="Search Chat Messages"
+            android:imeOptions="actionSearch"
             android:paddingLeft="@dimen/padding_moderate"
-            android:background="@color/colorPrimaryDark"
-            android:textSize="@dimen/text_size_normal"
             android:singleLine="true"
-            android:imeOptions="actionSearch"/>
+            android:textSize="@dimen/text_size_normal"
+            android:visibility="gone" />
+
+        <FrameLayout
```

Comment:
why?

Reply:
FrameLyout is added for fine blackish link below EditText to create a separation from RecyclerView